packages/wonder-blocks-clickable/src/components/__tests__/clickable.typestest.tsx

```diff
@@ -47,3 +47,18 @@ import Clickable from "../clickable";
 >
     {(_) => "Hello, world!"}
 </Clickable>;
+
+<Clickable beforeNav={() => Promise.resolve()}>
+    {(_) => "Hello, world!"}
+</Clickable>;
+
+<Clickable safeWithNav={() => Promise.resolve()}>
+    {(_) => "Hello, world!"}
+</Clickable>;
+
+<Clickable
+    beforeNav={() => Promise.resolve()}
+    safeWithNav={() => Promise.resolve()}
+>
+    {(_) => "Hello, world!"}
+</Clickable>;
```

Comment:
We should probably make either `href` or `onClick` required, but I've fiddled enough with the types for today and would like to move on to other things.